src/Groups/perm.jl

```diff
@@ -161,6 +161,13 @@ julia> length(moved_points(gen(s, 1)))
 ```
 """
 @gapattribute moved_points(x::Union{PermGroupElem,PermGroup}) = Vector{Int}(GAP.Globals.MovedPoints(GapObj(x)))
+"""
+    fixed_points(G::PermGroup)
+
+Return the set of fixed points of the permutation group `G`.
```

Comment:
This states that the function returns a "set", but it doesn't return a `Set` but a `Vector`. @fingolfin since you opened the corresponding issue: Do you want this function to return a `Set` or a `Vector`? Should we guarantee the vector to be sorted?

Reply:
A sorted vector, to match `moved_points`.

While we are it: @varuntrehan7 please also add a function `number_of_fixed_points` which is implemented by calling `number_of_moved_points` suitably.